File: OneChickenPerPerson.java
import java.util.Scanner;

public class OneChickenPerPerson {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int people = sc.nextInt();
		int chicken = sc.nextInt();
		
		int total = chicken - people;
		
		if(total == 1) {
			System.out.println("Dr. Chaz will have " + total + " piece of chicken left over!");
		} else if(total > 1 || total == 0) {
			  System.out.println("Dr. Chaz will have " + total + " pieces of chicken left over!");	
		} else if(total == -1) {
			total = total * -1;
			System.out.println("Dr. Chaz needs " + total + " more piece of chicken!");
		} else {
			total = total * -1;
			System.out.println("Dr. Chaz needs " + total + " more pieces of chicken!");
		}
	}
}

File: Python Solutions/OneChickenPerPerson.py
a,b = map(int, input().split())
c = abs(a-b)
if a == b+1:
    print('Dr. Chaz needs ' + str(c) + ' more piece of chicken!')
elif a > b:
    print('Dr. Chaz needs ' + str(c) + ' more pieces of chicken!')
elif a+1 == b:
    print('Dr. Chaz will have ' + str(c) + ' piece of chicken left over!')
elif b > a:
    print('Dr. Chaz will have ' + str(c) + ' pieces of chicken left over!')